bentoml/_internal/bento/build_config.py

```diff
@@ -110,28 +107,29 @@ def _envars_validator(
     attribute: "Attribute[t.Union[str, t.List[str], t.Dict[str, t.Any]]]",
     value: t.Union[str, t.List[str], t.Dict[str, t.Any]],
 ) -> None:
-    if isinstance(value, str):
-        if not os.path.exists(value) and os.path.isfile(value):
-            raise InvalidArgument(
-                f'`env="{value}"` is not a valid path. if `env` is parsed as a string, it should be a path to an ".env" file.'
-            )
-
-    envars_rgx = re.compile(r"(^[A-Z0-9_]+)(\=)(.*\n)")
-
     if isinstance(value, list):
-        for envar in value:
-            match = envars_rgx.match(envar)
-            if match is None:
-                raise InvalidArgument(f"{envar} doesn't follow format ENVAR=value")
-
-            k, _ = envar.split("=")
-            if not k.isupper():
+        for envar in value:  # pragma: no cover
```

Comment:
remove `pragma: no cover`?

Reply:
I have tests for this, but it doesn't seem to add it into coverage.

Reply:
could mean that this statement is never reached?

Reply:
If this is using branch coverage it might be that value never has length 0.